app/src/main/java/com/mensinator/app/settings/SettingsScreen.kt

```diff
@@ -83,6 +83,15 @@ object ResourceMapper {
     fun getStringResourceId(key: String): Int? {
         return resourceMap[key]
     }
+
+    @Composable
```

Comment:
```
    /**
     * - If key is unchanged, return the R.string value
     * - If key has changed (null), return user-set value
     */
    @Composable
    fun getStringResourceOrNew(key: String): String {
        val id = getStringResourceId(key)
        val text = id?.let { stringResource(id = id) } ?: key
        return text
    }
```

KDoc-enabled block comment (Use double `/**` to start that)

Reply:
Nice! I wasn't sure of the docstring/comment syntax in Kotlin, changed